### ISSUE TITLE
scrobble doesn't work

### ISSUE THREAD

**Issue by tomfrontman** [trakt-tools/universal-trakt-scrobbler#345]
Even though I have activated scrobble, it doesn't work, neither on Amazon Prime nor on Disney+. On netflix it "works", because to make it work I need to also have trakflix.

**Comment by MrMamen** [comment]
Are you sure you enabled scrobbling for these services in the settings?

**Comment by tomfrontman** [comment]
yes I'm sure of it. Furthermore, Netflix doesn't even work now, because trakflix stops working after 2/3 days.

**Comment by bbaraniec** [comment]
Same for me, only Netflix works.

**Comment by RJVMS** [comment]
Same for me, only seems to work on netflix

**Comment by MrN0b0dy98** [comment]
Seems to be a bug since July ([v0.3.0](https://github.com/trakt-tools/universal-trakt-scrobbler/releases/tag/0.3.0) release?)

[Netflix and Amazon Prime not scrobbling #329](https://github.com/trakt-tools/universal-trakt-scrobbler/issues/329)
[Disney+ not scrobbling #326](https://github.com/trakt-tools/universal-trakt-scrobbler/issues/326)

**Comment by tomfrontman** [comment]
Yes, it is the same bug. I have the v0.3.1 and the bug still there, so I think he still have to fix it

**Comment by pax0707** [comment]
Netflix is working for me.
D+ and MAX not.

**Comment by magnusssz** [comment]
disney+ still not working.
